src/altinn-app-frontend/src/features/form/containers/GroupContainer.tsx

```diff
@@ -124,6 +101,25 @@ export function GroupContainer({ id, container, components }: IGroupProps): JSX.
     [dispatch, id],
   );
 
+  const addButon = () => (
```

Comment:
Buton, is that french for something? :sweat_smile: :shrug:

Reply:
indeed 🥖 🍷 🇫🇷